src/tests/work-item/work-item-list/displayAndUpdateWorkItemDetails.spec.js

```diff
@@ -315,7 +316,8 @@ it('Verify how many work item type exists in drop down - desktop', function() {
           expect(page.workItemTitle(page.firstWorkItem)).toBe(workItemTitle);
           page.workItemViewId(page.workItemByTitle(workItemTitle)).getText().then(function (text) {
             var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
-            expect(detailPage.WorkItemTypeDropDownListCount()).toBe(7);
+            //Commenting type drop down related tests for issue 635
+            //expect(detailPage.WorkItemTypeDropDownListCount()).toBe(7);
```

Comment:
I think it's better to disable the whole spec and not just the return.
From line (old) 316 to (old) 320.